piDIOComm.c

```diff
@@ -118,38 +118,33 @@ INT32U piDIOComm_Config(uint8_t i8uAddress, uint16_t i16uNumEntries, SEntryInfo
 
 INT32U piDIOComm_Init(INT8U i8uDevice_p)
 {
+	u8 snd_len = sizeof(SDioConfig) - IOPROTOCOL_HEADER_LENGTH - 1;
 	u8 addr = RevPiDevice_getDev(i8uDevice_p)->i8uAddress;
+	UIoProtocolHeader *hdr;
+	u8 *snd_buf;
 	int ret;
-	SIOGeneric sResponse_l;
-	INT8U i, len_l;
+	int i;
 
-	pr_info_dio("piDIOComm_Init %d of %d  addr %d numCnt %d\n", i8uDevice_p, i8uConfigured_s,
-		    RevPiDevice_getDev(i8uDevice_p)->i8uAddress,
-		    i8uNumCounter[RevPiDevice_getDev(i8uDevice_p)->i8uAddress]);
+	pr_info_dio("piDIOComm_Init %d of %d  addr %d numCnt %d\n", i8uDevice_p,
+		i8uConfigured_s, RevPiDevice_getDev(i8uDevice_p)->i8uAddress,
+		i8uNumCounter[RevPiDevice_getDev(i8uDevice_p)->i8uAddress]);
```

Comment:
If you just use `addr` instead of `RevPiDevice_getDev(i8uDevice_p)->i8uAddress`, you can align the parameters neatly to the opening parenthesis without exceeding 80 chars per line.